draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2028,1226 +1972,1147 @@ uCDN-A                         dCDN-B                         dCDN-C
+    "type": "json"
+  }
+]
+]]></sourcecode>
 
-         <t>
-           Example JSON serialized UTCWindow object that describes a time window in New York time zone offset
-           UTC-05:00 from 02:30 01/01/2000 to 04:30 01/01/2000:
-         </t>
-        <sourcecode type="json" name="utcwindow-object-2"><![CDATA[
-{
-  "start": "2000-01-01T02:30:00.00-05:00",
-  "end": "2000-01-01T04:30:00.00-05:00"
-}
-         ]]></sourcecode>
-        </section>
-   </section>
-   <section anchor="execution-policy" title="ExecutionPolicy Extension">
-   <t>
-      Unless specified otherwise, dCDN is at liberty to decide how to choose trigger commands for execution from all pending commands, whether to process trigger commands sequentially or in parallel, immediately upon acceptance, or with a delay in batches. uCDN may wish to control trigger processing in more detail, including the order of execution, dependencies, and concurrency.
-   </t>
-   <t>
-   Example use cases:
-      <list style="symbols">
-      <t>
-      Priority:  uCDN may have multiple trigger commands in "pending" and/or "active" mode. For example, trigger commands with policy constraints, a large number of content objects affected, or other dCDN business logic may take a long time to execute.
-      uCDN may wish to prescribe the order in which dCDN picks up its trigger commands for execution from the "pending" queue, by indicating a relative priority of each trigger. The priority would affect the selection of trigger commands specific to the requesting uCDN. dCDN may separately prioritize triggers from multiple uCDNs subject to its business logic. 
+   <t>Embedded JSON-encoded object list:</t>
+   <sourcecode type="json" name="embedded-json-object-list"><![CDATA[
+[
+  {
+    "data": [
+      {
+        "href": "https://example.com/hls/35cdc008/index.m3u8",
+        "type": "hls"
+      },
+      {
+        "href": "https://example.com/dash/35cdc008/main.mpd",
+        "type": "dash"
+      },
+      {
+        "href": "https://example.com/img/35cdc008/thumb-l.jpg",
+        "size": 10260
+      },
+      {
+        "href": "https://example.com/img/35cdc008/thumb-s.jpg",
+        "size": 1453
+      }
+    ],
+    "type": "json"
+  }
+]
+]]></sourcecode>
 
-      Multiple priority-related use cases exist: 
-         <list style="symbols">
-         <t> uCDN needs to introduce an urgent "purge" or "invalidate" trigger into an existing queue of trigger commands to correct wrong versions of content objects published by it</t>
-         <t> uCDN needs to indicate which content objects should be prepositioned, purged or invalidated first, for example prepositioning newer released content before prepositioning updates to an existing catalog</t>
-         </list>
-      </t>
+   <t>Embedded HLS manifest:</t>
+   <t>Please note that some lines in the example are wrapped for clarity.</t>
+   <sourcecode type="json" name="embedded-hls-manifest"><![CDATA[
+[
+  {
+  "data": "#EXTM3U\n
```

Comment:
The spacing of this example got messed up during the editing process.  The content should be flush with the first column, and the rest of the JSON elements should be indented more.

Reply:
ACK